src/redux/slices/events/middleware.ts

```diff
@@ -53,6 +54,20 @@ const getSingleEvent = (id: string) => async (dispatch: AppDispatch) => {
   }
 }
 
+const createEvent = (formData: ICreateEventParams) => async (dispatch: AppDispatch) => {
+  try {
+    dispatch(setSingleEventLoading(true))
+
+    await API.events.createEvent(formData)
+
+    // dispatch(setSingleEventData(response.data.data))
```

Comment:
why comment? get data and set it in redux store, get it in the component and show it

Reply:
this is not get data this is call request for create events